test/prob/test_fixture_ccdf_log.hpp

```diff
@@ -351,11 +354,14 @@ class AgradCcdfLogTestFixture : public ::testing::Test {
         << "Number of finite diff gradients and calculated gradients must "
            "match -- error in test fixture";
     for (size_t i = 0; i < finite_dif.size(); i++) {
-      EXPECT_NEAR(finite_dif[i], gradients[i], 1e-4)
-          << "Comparison of finite diff to calculated gradient failed for i="
-          << i << ": " << parameters << std::endl
-          << "  finite diffs: " << finite_dif << std::endl
-          << "  grads:        " << gradients;
+      std::stringstream stream;
+      stream << "Comparison of finite diff to calculated gradient failed for i="
+             << i << ": " << parameters << std::endl
+             << "  finite diffs: " << finite_dif << std::endl
+             << "  grads:        " << gradients;
+
+      stan::test::expect_near_rel(stream.str(), finite_dif[i], gradients[i],
+                                  stan::test::relative_tolerance(1e-4, 1e-7));
```

Comment:
1e-4 relative error (this is what the unit tests use for gradients, see here: https://github.com/stan-dev/math/blob/develop/test/unit/math/ad_tolerances.hpp)

I think I used 1e-7 for the minimum error tolerance cause 1e-8 didn't work for some function. I'm fuzzy on this.